low-level/files/opt/seagate/sspl/setup/resource_map/lr2/server.py

```diff
@@ -461,3 +468,56 @@ def get_nw_status(self, nw_interface, interface):
             nw_cable_conn_status = "UNKNOWN"
             logger.exception(self.log.svc_log(err))
         return nw_status, nw_cable_conn_status
+
+    def get_disks_info(self):
+        """Update and return server drive information in specific format."""
+        units_factor_GB = 1000000000
+        overall_usage = {
+                "totalSpace": f'{int(psutil.disk_usage("/")[0])//int(units_factor_GB)} GB',
+                "usedSpace": f'{int(psutil.disk_usage("/")[1])//int(units_factor_GB)} GB',
+                "freeSpace": f'{int(psutil.disk_usage("/")[2])//int(units_factor_GB)} GB',
+                "diskUsedPercentage": psutil.disk_usage("/")[3],
+            }
+        disks = []
+        for disk in Disks().get_disks():
+            uid = disk.path if disk.path else disk.id
```

Comment:
>https://jts.seagate.com/secure/attachment/467631/disks.json
        "uid": "ST1000NM0055-1V410C-ZC236QHZ",

we need disk_\<slot> or disk dev-id with path kind of uid, serial number can be in specifics

Reply:
Used by-path
ex. pci-0000:00:11.5-ata-2.0

Reply:
>ex. pci-0000:00:11.5-ata-2.0

thats how it looks for a disk ?

Reply:
We are using same in sensor.

Reply:
hmm so in sensor also we would need the change. keep TODO